tests/Unit/Endpoints/MerchantsTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Unit\Endpoints;
+
+use Alma\API\Endpoints\Merchants;
+use Alma\API\Exceptions\RequestException;
+use Alma\API\Request;
+use Alma\API\Response;
+use Mockery;
+use PHPUnit\Framework\TestCase;
+
+class MerchantsTest extends TestCase
+{
+	const URL = "https://www.example.com/integrations/configurations";
+
+	public function setUp(): void
+	{
+		$this->merchantEndpoint = Mockery::mock(Merchants::class)->makePartial();
```

Comment:
We don't want to remove Mockery and use native mock ?

Reply:
When we want to test an endpoint, I'm note sure we can do it without a makePartial

Reply:
I saw their is a similar like Mockery with a native PhpUnit, but we can explore it for a next PR :)